app/forms/hub/client_filter_form.rb

```diff
@@ -0,0 +1,16 @@
+module Hub
+  class ClientFilterForm < Form
+    attr_accessor :sort_column, :sort_order, :_filtered_and_sorted_clients
+
+    def filtered_and_sorted_clients(clients)
+      clients = clients.after_consent
```

Comment:
I wonder if this should be moved into the Controllers, because it feels like a high level scope rather then a filter based on inputs